schema/filter_artworks.js

```diff
@@ -23,10 +23,16 @@ export const FilterArtworksType = new GraphQLObjectType({
       type: GraphQLInt,
       resolve: ({ aggregations }) => aggregations.total.value,
     },
+    followed_artists_total: {
+      type: GraphQLInt,
+      resolve: ({ aggregations }) => aggregations.followed_artists.value,
+    },
     aggregations: {
       type: new GraphQLList(ArtworksAggregationResultsType),
-      resolve: ({ aggregations }) =>
-        map(omit(aggregations, ['total']), (counts, slice) => ({ slice, counts })),
+      resolve: ({ aggregations }) => {
+        const whitelistedAggregations = omit(aggregations, ['total', 'followed_artists']);
+        return map(whitelistedAggregations, (counts, slice) => ({ slice, counts }));
+      },
```

Comment:
This field go really do with a `description`, I can’t really tell what an ‘aggregation’ vs ‘hits’ is.